packages/property-projector/src/components/ui/colour-picker.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react'
+import { Input } from '@reapit/elements'
+import { SketchPicker } from 'react-color'
+
+export type ColourPickerProps = {
+  id: string
+  name: string
+  placeholder: string
+  labelText: string
+  colour: string
+  onChange?: {
+    (colour: string): void
+  }
+}
+
+const ColourPicker: React.FC<ColourPickerProps> = props => {
+  const { id, name, placeholder, labelText, colour, onChange } = props
+
+  const [showColourPicker, setShowColourPicker]: any = useState(false)
```

Comment:
Syntax for use state is `const [showColourPicker, setShowColourPicker] = useState<boolean>(false)`